node/blockchain/management/commands/generate_node_signing_key.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from node.blockchain.constants import NODE_SIGNING_KEY
+from node.core.utils.cryptography import generate_key_pair
+
+from .utils import set_env_key
+
+
+class Command(BaseCommand):
+    help = 'Generate node signing (private) key'  # noqa: A003
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-w',
+            '--write',
+            action='store_true',
+            default=False,
+            help='Save generated Node Signing Key to .env file with TNB_NODE_SIGNING_KEY key. '
+            'Default: False'
+        )
+        parser.add_argument(
+            '-f',
+            '--force',
+            action='store_true',
+            default=False,
+            help='Rewrite existed value in .env file. Default: False'
+        )
+        parser.add_argument(
+            '-c',
+            '--config-path',
+            type=str,
+            default=settings.ENV_CONFIG_PATH,
+            help=f'Path to .env file. Default: {settings.ENV_CONFIG_PATH}'
+        )
+
+    def handle(self, config_path: str, write: bool, force: bool, *args, **options):
+        node_signing_key = generate_key_pair().private
+
+        if write is True:
```

Comment:
I'd rather just print `node_signing_key` to stdout always. Then we can redirect it to anywhere we want. No need for dealing with .env paths, storing env var names in separate variables, etc.

Please, see how it is implemented in thenewboston-node

Reply:
Both scripts return generated values to stdout only.